docs/getting_started.md

```diff
@@ -0,0 +1,124 @@
+# Getting Started
+
+## System Prerequisites
+
+DAPper is written in Rust and requires the Rust toolchain. It is recommended to use the latest stable Rust version (1.70.0 or newer). You can install through your package manager or through [rustup.rs](https://www.rust-lang.org/tools/install).
+
+## Dataset Setup
+DAPper uses pre-built datasets that map file names to packages for various ecosystems. These datasets are hosted on Hugging Face and need to be downloaded and placed in the appropriate XDG directory.
+
+### Available Datasets
+The following datasets are available at https://huggingface.co/dapper-datasets:
+
+dapper-datasets/pypi - Python Package Index mappings
+dapper-datasets/debian-bookworm - Debian Bookworm packages
+dapper-datasets/debian-bullseye - Debian Bullseye packages
+dapper-datasets/debian-buster - Debian Buster packages
+dapper-datasets/ubuntu-noble - Ubuntu Noble packages
+dapper-datasets/ubuntu-jammy - Ubuntu Jammy packages
+dapper-datasets/ubuntu-focal - Ubuntu Focal packages
+dapper-datasets/NuGet-dataset - .NET NuGet packages
+
+### Set up 
+1. Create data directory: 
+Linux/macOS:
+```bash
+mkdir -p ${XDG_DATA_HOME:-$HOME/.local/share}/dapper/
+```
```

Comment:
I feel like on macOS it's not `$XDG_DATA_HOME/.local/share/dapper/`, but `$HOME/Library/Application Support/dapper`